wireless_modules/mqtt_test/main.py

```diff
@@ -0,0 +1,108 @@
+import time
+import machine
+import config
+import ubinascii
+import fake
+from machine import Pin
+
+# Use micro pip to install packages
+import upip
+upip.install("micropython-umqtt.simple2")
+from umqtt.simple import MQTTClient
+
+
+# Define module number and MQTT topics
+MODULE_NUM = "1"
+FAKE_MODULE = fake.module(MODULE_NUM)
+
+PUB_DATA = b'/v3/wireless-module/{}/data'.format(MODULE_NUM)
+PUB_BATTERY = b'/v3/wireless-module/{}/battery'.format(MODULE_NUM)
+PUB_LOW_BATTERY = b'/v3/wireless-module/{}/low-battery'.format(MODULE_NUM)
+
+SUB_LED = b'/v3/wireless-module/{}/LED'.format(MODULE_NUM)
+SUB_TOPICS = [SUB_LED]
+
+# Define built in LED
+builtin_LED = Pin(2, Pin.OUT)
```

Comment:
Is it possible to use the same `builtin_LED` definition here and in `boot.py`? Maybe it could be stored in config